src/connectors/pagination/connectPagination.js

```diff
@@ -180,13 +180,9 @@ export default function connectPagination(renderFn, unmountFn = noop) {
       },
 
       getWidgetSearchParameters(searchParameters, { uiState }) {
-        const uiPage = uiState.page;
+        const page = uiState.page ? uiState.page - 1 : 0;
```

Comment:
It seamed to be there before but can you explain why we set the page to `page - 1` when there's a UI state for page? Do we considerer `page: 1` to be the first page (i.e. `0` in code)?

If so, it's worth adding a comment.

Reply:
Yes it is. At the API/`SearchParameters` level page is indexed on 0. Inside the `uiState` it's indexed on 1. Most of the time users expect the first page to be `1` not `0`, especially inside their URL. That's why we have this (see the [`getWidgetState`](https://github.com/algolia/instantsearch.js/blob/299ffe28006deb516bdc969da8089e703623be6b/src/connectors/pagination/connectPagination.js#L158-L165)).